### ISSUE TITLE
Facing desktop screen stuck issue to take remote, below error is coming on on Console

### ISSUE THREAD

**Issue by AnjaniKumar15** [Ylianst/MeshCentral#1447]
uncaughtException1: Error: timers.onElapsed() callback handler on '()'  => TypeError: cannot delete property 'dest' of undefined.

Using Version 0.5.50 

I want to switch to stable version. but not able to switch from 0.5.50 to 0.5.11 stable.

**Comment by krayon007** [comment]
What os are you using?

**Comment by AnjaniKumar15** [comment]
Ubuntu 16.

**Comment by krayon007** [comment]
When you connect, does it say webrtc next to the connect button?

**Comment by pczolee** [comment]
I have the same problem. I reverted to the stable, but that one is a bit old, so would be fantastic if we can have a new working version.

And just a suggestion. Can we have an option to set which version we want to update/downgrade?

Thanks

**Comment by uldiseihenbergs** [comment]
@pczolee you can install every one version - just use a tag, for example `npm install meshcentral@0.5.47`

**Comment by AnjaniKumar15** [comment]
I am not using WebRTC rightnow. Earlier when I had just started using Meshcentral that time I was faced issues with WebRTC like screen stuck, so I had disabled this setting from config.

**Comment by krayon007** [comment]
> I have the same problem. I reverted to the stable, but that one is a bit old, so would be fantastic if we can have a new working version.
> 
> And just a suggestion. Can we have an option to set which version we want to update/downgrade?
> 
> Thanks

What OS are you running?

**Comment by AnjaniKumar15** [comment]
> When you connect, does it say webrtc next to the connect button?

No, its not, Actually I am not using WebRTC rightnow. Earlier when I had just started using Meshcentral that time I was faced issues with WebRTC like screen stuck, so I had disabled this setting from config

**Comment by krayon007** [comment]
I happen to have a Ubuntu 16 test machine, so I'll see if I can reproduce it... I haven't had any issues with KVM on linux so far...

**Comment by pczolee** [comment]
> @pczolee you can install every one version - just use a tag, for example `npm install meshcentral@0.5.47`

That's fine, thanks! But is we can do this on the web, that's would be fantastic!

**Comment by pczolee** [comment]
> > I have the same problem. I reverted to the stable, but that one is a bit old, so would be fantastic if we can have a new working version.
> > And just a suggestion. Can we have an option to set which version we want to update/downgrade?
> > Thanks
> 
> What OS are you running?

On the serves side I am using ubuntu 12.04, with node 10.17 . The client is ubuntu 18.04 with firefox 76. Now I tried the 0.5.54, but the issue is same :(

**Comment by pczolee** [comment]
Another thing. It seems, that the remote machines which are running windows, these are working, but which have linux these have this screen issues.
Now I tried to downgrade one by one version back to find which one was the last version which running well. So I found that the 0.5.36 is the last which running without problem, after that there is the screen issue!

**Comment by pczolee** [comment]
Are there any way how can I help to debug and solve the problem?
